app/ovpn/get-ovpn-files.sh

```diff
@@ -2,19 +2,6 @@
 
 . /app/date.sh --source-only
 
-# check if the file exists
-if [ -f ${OVPN_CONFIG_DIR}/ovpn.zip ]; then
-  #the file exists continue checking if its older than two hours.
-  if test `find ${OVPN_CONFIG_DIR}/ovpn.zip -mmin +${REFRESH_TIME}`; then
-    download_files
-  else
-    echo "$(adddate) INFO: Skipping downloading OVPN files - as they are not older than two hours."
-  fi
-else
-  #the files don't exists contiue to download and extract them.
-  download_files
-fi
-
 function download_files() {
```

Comment:
It should either be
```
download_files() {
```
or
```
function download_files {
```

See https://linuxize.com/post/bash-functions/

Reply:
Should be fixed in the newest commit